src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -186,6 +189,14 @@ def put(self, request, organization, version):
                         datetime=release.date_released,
                     )
 
+            analytics.record(
+                "release.set_commits",
```

Comment:
@NisanthanNanthakumar so we are emitting this event even if we don't actually associate commits with this release?